### PR TITLE
Minor update

### PR DESCRIPTION
**Pull request by adnan-khattak**

### CODE REVIEW

File: src/complaints/InitiatedComplaint.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      },
+      image: {
+        width: '100%',
+        height: '100%',
+        resizeMode: 'contain',
+        borderRadius: 8,
+      },    
+});
+
+export default InitiatedComplaint
```
Comment (Zakarya525) [inline src/complaints/InitiatedComplaint.js]: **Hi there! I'd like to propose a suggestion for organizing styles within the complaints directory.**

**Could we consider centralizing all complaint-related styles into a single file, such as styles.js, and then import specific styles from that file as needed within individual components?**

Here are the potential benefits of this approach:

- Enhanced maintainability: Congregating styles in one place simplifies tracking and updating them.
- Improved organization: A centralized file fosters a clear structure and reduces style scattering.
- Potential performance gains: Reusing styles from a single file can minimize redundant CSS generation.
- Modularity and reusability: Isolating styles promote their easy transfer between components.
- Enhanced readability: A dedicated styles file facilitates code comprehension and visual consistency.

Implementation example:

1. Create styles.js: 
// styles.js
```
export const complaintHeading = `
  /* Styles for complaint headings */
`;

export const complaintName = `
  /* Styles for complaint body text */
`;
// ... other complaint-related styles
```
2. Import in components:
// ClosedComplaint.js
```
import { complaintHeading, complaintName } from './styles.js';

export default ClosedComplaint = () => {
  return (
    <div className={complaintHeading}>Complaint Form</div>
    <Text style={styles.complaintName}>{ComplaintData.complaintName}</Text>
      {/* Complaint content */}
    </div>
  );
}
```

**I'm open to discussing this further if you have any questions or concerns!**

File: src/screens/RecievedComplaint.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+});
+
+export default RecievedComplaint;
```
Comment (Zakarya525) [inline src/screens/RecievedComplaint.js]: I'd like to propose a structural refinement for screens within the **complaints** directory:

- If a file represents a full-fledged screen, I recommend relocating it to a dedicated screens directory within the complaints folder. This aligns with best practices for organizing screens and enhances code clarity.

- If a file contains reusable components specific to the complaints functionality, consider moving it to a **components** directory within complaints. This promotes modularity and reusability.

By adhering to these structural guidelines, we can foster a more organized and maintainable codebase 🙂

**Review by Zakarya525 — COMMENTED** [review]
I have added some feedback, please review

File: src/complaints/InitiatedComplaint.js (hunk 1 above)
Comment (adnan-khattak) [reply src/complaints/InitiatedComplaint.js]: Okay That`s great to hear soon that should be implemented.

File: src/screens/RecievedComplaint.js (hunk 2 above)
Comment (adnan-khattak) [reply src/screens/RecievedComplaint.js]: Hmm this is a full-fledged screen and it contain reusable component as well as  it has two main screen i.e
1) Pending Complaint
2) Recieved Complaint
and further individual complaint status 
i got your feedback and soon i will arrange the file like your suggestion

**Comment by Zakarya525** [comment]
after changes and merge, you can close this PR safely

File: src/screens/RecievedComplaint.js (hunk 2 above)
Comment (adnan-khattak) [reply src/screens/RecievedComplaint.js]: @Zakarya525 plz review the structure now it is updated

**Pull request merged**